GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -334,7 +334,7 @@ public void Add(GitRevision revision, bool insertWithMatch = false, int insertRa
                 {
                     foreach (var parentId in parents)
                     {
-                        if (_revisionGraph.TryGetNode(parentId, out RevisionGraphRevision parentRev))
+                        if (_revisionGraph.TryGetNode(parentId, out var parentRev))
```

Comment:
Why?

Reply:
This is being changed from var to the actual type.
I believe this was changed by analyzers with what was possible a couple of years ago, but no generic cleanup.

Reply:
Here's the rule I'm referring to: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md#c-coding-style point 10.

We still need to finalise https://github.com/gitextensions/gitextensions/pull/8843 (with additional tweaks, if necessary) and update [our coding style guide](https://github.com/gitextensions/gitextensions/wiki/Guidelines%3A-Coding-Style).